### DISCUSSION TITLE
Why are installed profiles different from the repo profiles?

### DISCUSSION THREAD

**Discussion by stargazer1984** [roddhjav/apparmor.d#588] [Q&A] [answered]
Following the install instructions I build apparmor.d from this AUR package https://aur.archlinux.org/apparmor.d-git.git
As far as I can see, this simply installs the most recent git version.

However, when I take a look at the installed profiles, they are very different from the profiles in the git repo.

For example this line  https://github.com/roddhjav/apparmor.d/blob/db6c94ba5ad97112bc577cb66c2e1fa66df83a29/apparmor.d/profiles-g-l/git#L69 uses the `rPx` mode, while the installed profiles all have `rpx` 
The lower case p apparently is not recommended:

> px does not scrub the environment of variables such as LD_PRELOAD. As a result, the calling domain may have an undue amount of influence over the called item.
https://www.novell.com/documentation/apparmor/apparmor201_sp10_admin/data/bx5bmls.html

Why are the installed files different the then repo files? Is this intended?

**Comment by beroal** [comment, marked as answer]
Regarding the transformation of `rPx` into `rpx`, see #344.

**Reply by stargazer1984** [reply]
I assume this is also the reason for converting rCx to rcx ?

**Reply by roddhjav** [reply]
Yes it is. It should be fixed at some point. However, it is currently not a priority (current priority is on profile testing and stabilisation)
